Gui/GeometryGUI/Area/AreaView.java

```diff
@@ -0,0 +1,141 @@
+	private AreaTriangleComponent triPaint = new AreaTriangleComponent();
+	private AreaSquareComponent squPaint = new AreaSquareComponent();
+	
+	public AreaView() {
+		setTitle("Geometry- Area Calculator");
+		setSize(800, 600);
+		setDefaultCloseOperation(EXIT_ON_CLOSE);
+		
+		//main panel declaration and layout set
+		JPanel panel = new JPanel();
+		panel.setLayout(new GridLayout(4,3));
+		//four input panels declaration and layout set
+		JPanel recPanel = new JPanel();
+		recPanel.setLayout(new GridLayout(2, 2));
+		JPanel cirPanel = new JPanel();
+		cirPanel.setLayout(new GridLayout(1, 2));
+		JPanel triPanel = new JPanel();
+		triPanel.setLayout(new GridLayout(2, 2));
+		JPanel squPanel = new JPanel();
+		squPanel.setLayout(new GridLayout(1, 2));
+		
+		//first row
+		panel.add(rectangle);
+		panel.add(recPanel);
+		panel.add(recPaint);
+		//second row
+		panel.add(circle);
+		panel.add(cirPanel);
+		panel.add(cirPaint);	
+		//third row
+		panel.add(triangle);
+		panel.add(triPanel);
+		panel.add(triPaint);	
+		//fourth row
+		panel.add(square);
+		panel.add(squPanel);
+		panel.add(squPaint);	
+		
+		//rectangle input panel
+		recPanel.add(recHeightLabel);
+		recPanel.add(recHeight);
+		recPanel.add(recWidthLabel);
+		recPanel.add(recWidth);
+		recHeight.addPropertyChangeListener("value", this::recRepaint);
+		recWidth.addPropertyChangeListener("value", this::recRepaint);
+		
+		//circle input panel
+		cirPanel.add(cirRadiusLabel);
+		cirPanel.add(cirRadius);	
+		cirRadius.addPropertyChangeListener("value", this::cirRepaint);
+		
+		//triangle input panel
+		triPanel.add(triLengthLabel);
+		triPanel.add(triLength);
+		triPanel.add(triHeightLabel);
+		triPanel.add(triHeight);
+		triLength.addPropertyChangeListener("value", this::triRepaint);
+		triHeight.addPropertyChangeListener("value", this::triRepaint);
+		
+		//triangle input panel
+		squPanel.add(squSideLabel);
+		squPanel.add(squSide);		
+		squSide.addPropertyChangeListener("value", this::squRepaint);
+		
+		add(panel);
+	}
+
+	public void recRepaint(PropertyChangeEvent event) {
+		double height = Double.parseDouble(recHeight.getText());
+		double width = Double.parseDouble(recWidth.getText());
+		recPaint.setRectangleHeight(height);
+		recHeight.setValue(height);
+		recPaint.setRectangleWidth(width);
+		recWidth.setValue(width);
+		recPaint.repaint();
+	}
+	
+	public void cirRepaint(PropertyChangeEvent event) {
+		double radius = Double.parseDouble(cirRadius.getText());
+		cirPaint.setRadius(radius);
+		cirRadius.setValue(radius);
+		cirPaint.repaint();
+	}
+	
+	public void triRepaint(PropertyChangeEvent event) {
+		double height = Double.parseDouble(triHeight.getText());
+		double width = Double.parseDouble(triLength.getText());
+		triPaint.setHeight(height);
+		triHeight.setValue(height);
+		triPaint.setLength(width);
+		triLength.setValue(width);
+		triPaint.repaint();
+	}
+	
+	public void squRepaint(PropertyChangeEvent event) {
```

Comment:
write out the whole method name, `squareRepaint`
Do the same for the other methods above.

Reply:
done